data/base.go

```diff
@@ -0,0 +1,286 @@
+package data
+
+import (
+	"fmt"
+	"reflect"
+	"time"
+
+	valid "github.com/tidepool-org/platform/Godeps/_workspace/src/gopkg.in/bluesuncorp/validator.v8"
+	"github.com/tidepool-org/platform/Godeps/_workspace/src/labix.org/v2/mgo/bson"
+
+	"github.com/tidepool-org/platform/validate"
+)
+
+//used for all data types
+var validator = validate.NewPlatformValidator()
```

Comment:
Ahhh, THIS is where `validator` comes from!  Totally not clear in basal.go.  Perhaps wrap this in a getter like `GetPlatformValidator` so it is obvious in other files?